### PR TITLE
Rabo should give a speech whenever Cutefish is mentioned.

### PR DESCRIPTION
**Pull request by muhdsalm**
I *may* have gone too far with the rabofication, and I'm new to this, so you may have to change/suggest changes to it. Also, if you have a personal rant, feel free to do that instead. I don't have feelings towards Cutefish, but I know you guys do. (Maybe Eri should do this -> https://discord.com/channels/957104883490775061/957432873701503047/1225060755926159511)

### CODE REVIEW

File: src/modules/cutefish-info.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import client from '../client';
+import { Events, Message } from 'discord.js';
+
+const CUTEFISH_MESSAGE = `Ok, wisten up, you! Cutefish used to be part of Ultramarine, until the devewoper just weft, cwosed the website, and weft the GitHub. There's no wevival of Cutefish that wasts wong, and we won't mantain it. If you can pwove to use that a wevival of Cutefish will wast wong, we'll think about it. Now scoot!
+Sowwy if you were just saying the word in nowmal convewsation. Keep gowing :3`
+
+const containsWord = (msg: Message, word: string): boolean => {
```
Comment (lleyton) [inline src/modules/cutefish-info.ts:7]: Please don't duplicate this function, if you need to use it from multiple files, put it into `util.ts` and import it.

File: src/modules/cutefish-info.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import client from '../client';
+import { Events, Message } from 'discord.js';
+
+const CUTEFISH_MESSAGE = `Ok, wisten up, you! Cutefish used to be part of Ultramarine, until the devewoper just weft, cwosed the website, and weft the GitHub. There's no wevival of Cutefish that wasts wong, and we won't mantain it. If you can pwove to use that a wevival of Cutefish will wast wong, we'll think about it. Now scoot!
```
Comment (lleyton) [inline src/modules/cutefish-info.ts:4]: looks amazing :3

**Review by lleyton — CHANGES_REQUESTED** [review]

File: src/modules/cutefish-info.ts (hunk 2 above)
Comment (muhdsalm) [reply src/modules/cutefish-info.ts:4]: Good to know!

File: src/modules/cutefish-info.ts (hunk 1 above)
Comment (muhdsalm) [reply src/modules/cutefish-info.ts]: Done!

**Review by lleyton — APPROVED** [review]

**Pull request merged**